src/domain/creator/data/create.ts

```diff
@@ -1,19 +1,24 @@
 
 import database, { RecordData } from '^/integrations/database/module';
+import { generateId } from '^/integrations/utilities/crypto';
 
 import { RECORD_TYPE } from '../definitions/constants';
 import CreatorData from './CreatorData';
 
 export default async function create(fullName: string, nickname: string, email: string, portrait?: string): Promise<CreatorData>
 {
-    const joinedAt = new Date();
+    const id = generateId();
+    const now = new Date();
+
+    const joinedAt = now.toISOString();
```

Comment:
```suggestion
    const now = new Date();

    const id = generateId();
    const joinedAt = now.toISOString();
```

Moved the `id` value down to be a part of the record data group.